src/main/java/org/cactoos/map/MapEnvelope.java

```diff
@@ -124,7 +126,14 @@ public final void clear() {
     public final String toString() {
         return new StringBuilder()
             .append('{')
-            .append(new TextOf(this.entrySet()).toString())
+            .append(
```

Comment:
@DmitryBarskov Why use `Concatenated` and `StringBuilder` together?

Reply:
Do I do it right now?

Reply:
@DmitryBarskov yes you should fix the issue the reviewer found, I think here he is hinting at the fact there is useless extra code and that it could be simplified… not 100% sure :)

Reply:
@victornoel  Yes, I got it, thank you. Just wanted to draw attention to the changes.